### PR TITLE
Courses for semester 2020-2021 Spring updated

### PR DESCRIPTION
**Pull request by debajyotidasgupta**
This Pull request addresses the issue [Issue #12](https://github.com/metakgp/what-slot/issues/12). Following are the changes made in the following pull request.
* The steps mentioned in the [README.md](https://github.com/metakgp/what-slot#updating-for-new-semester) were followed for updating the courses
* The semester has been changed to SPRING 2020-2021.
@thealphadollar, please look into the changes made and inform if some improvement is needed in this PR. Thanks!

### CODE REVIEW

File: courses.json [hunk 1]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+[]
```
Comment (thealphadollar) [inline courses.json]: Hey @debajyotidasgupta I can see that the courses file is empty. We use this file for displaying the slots, courses, etc.

Can you please check once more if you used the right credentials?

**Review by thealphadollar — CHANGES_REQUESTED** [review]
I feel something went wrong, can you please have a look?

File: fetch.py [hunk 2]
```diff
@@ -23,6 +28,7 @@ def getData(dep):
 
 	courses = []
 
+	print(response.text)
```
Comment (thealphadollar) [inline fetch.py]: @debajyotidasgupta Do you think we should remove this?

Comment (debajyotidasgupta) [reply fetch.py]: Yeah okay, I will

**Comment by debajyotidasgupta** [comment]
@thealphadollar I have removed the print line. The readme file looks good?

**Review by thealphadollar — APPROVED** [review]
Thanks a lot @debajyotidasgupta for updating Whatslot for the semester.

**Comment by debajyotidasgupta** [comment]
Thanks, @thealphadollar !!

**Pull request merged**